.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,37 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: master
```

Comment:
```suggestion
    branches: $default-branch
```

Reply:
`$default-branch` is not a valid string here. It's just a placeholder that you can put in templates and that gets replaced by github when you add the template to a repo.